client/package.json

```diff
@@ -121,12 +123,21 @@
     "validator": "13.6.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.13.16",
+    "@storybook/addon-actions": "^6.2.9",
+    "@storybook/addon-essentials": "^6.2.9",
+    "@storybook/addon-links": "^6.2.9",
+    "@storybook/addon-postcss": "^2.0.0",
+    "@storybook/react": "^6.2.9",
     "@testing-library/jest-dom": "5.12.0",
     "@testing-library/react": "11.2.6",
     "autoprefixer": "9.8.6",
+    "babel-loader": "^8.2.2",
```

Comment:
All of the new dependencies were added automatically by Storybook, except for `@storybook/addon-postcss`.